docs/cars.py

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env python3
+import jinja2
+import os
+from collections import defaultdict
+from sortedcontainers import SortedList
+
+from common.basedir import BASEDIR
+from common.params import Params
+from selfdrive.car.car_helpers import interfaces, get_interface_attr
+from selfdrive.car.docs import Tier, Column, StarColumns, MAKES_GOOD_STEERING_TORQUE, get_footnote, get_star_icon
+from selfdrive.car.hyundai.radar_interface import RADAR_START_ADDR as HKG_RADAR_START_ADDR
+from selfdrive.test.test_routes import non_tested_cars
+
+
+def get_all_footnotes():
+  all_footnotes = {}
+  i = 1
+  for _, footnotes in get_interface_attr("Footnote").items():
+    if footnotes is not None:
+      for footnote in footnotes:
+        all_footnotes[footnote] = i
+        i += 1
+  return all_footnotes
+
+
+ALL_FOOTNOTES = get_all_footnotes()
+
+
+class Car:
+  def __init__(self, car_info, CP):
+    self.make, self.model = car_info.name.split(' ', 1)
+    self.row, star_count = self.get_row(car_info, CP)
+    self.tier = {5: Tier.GOLD, 4: Tier.SILVER}.get(star_count, Tier.BRONZE)
+
+  def get_row(self, car_info, CP):
+    # TODO: add YouTube videos
+    # Some minimum steering speeds are not yet in CarParams
+    min_steer_speed = CP.minSteerSpeed
+    if car_info.min_steer_speed is not None:
+      min_steer_speed = car_info.min_steer_speed
+      assert CP.minSteerSpeed == 0, "Minimum steer speed set in both CarInfo and CarParams for {}".format(CP.carFingerprint)
+
+    min_enable_speed = CP.minEnableSpeed
+    if car_info.min_enable_speed is not None:
```

Comment:
add an assert to catch when both are set

Reply:
See comment above, the Prius V sets stop and go to True for when the user installs a sDSU, but normally doesn't have ACC from 0.